### DISCUSSION TITLE
OCR via Google's Cloud Vision API

### DISCUSSION THREAD

**Discussion by voska7** [SubtitleEdit/subtitleedit#6359] [Q&A] [answered]
google cloud vision api method giving error time to time. here is the screenshot. if I click pause and start again, it works.

![image](https://user-images.githubusercontent.com/42012426/197991578-54f4af76-7a17-4d80-962d-92e32d86bd9f.png)

**Comment by voska7** [comment]
I don't know may be it's my internet connection/ISP issue. i asked another friend who has very fast internet connection. for whom such error did not show.

**Comment by niksedk** [comment, marked as answer]
Enable billing. See https://cloud.google.com/vision/docs/before-you-begin

**Comment by voska7** [comment]
yes i did which is why some part of line converted. i think it's my internet connectivity issue.

**Comment by Gatorman3385** [comment]
Are we able to train the API in order to fix mistakes?
